NEWS.md

```diff
@@ -20,6 +20,8 @@ to make the default behavior consistent. (#87)
 
 ## Documentation
   - New section in programming strategy regarding comments (#71)
+  - Removed requirement to add `@author` tags to code scripts from programming 
+  strategy, as we will no longer be tracking authors. (#206)
```

Comment:
```suggestion
  - Removed requirement to add `@author` tags to code scripts from programming 
  strategy, as we will only be tracking authors in the DESCRIPTION file. (#206)
```

Reply:
Ah I see what you meant by your original comment now, thanks.